### PR TITLE
Fixing setting of node resources in ES CR

### PR DESCRIPTION
**Pull request by ewolinetz**
Resolves https://bugzilla.redhat.com/show_bug.cgi?id=1671276

### CODE REVIEW

**Comment by ewolinetz** [comment]
/hold
I may need to retest this after #95

File: pkg/k8shandler/logstore.go [hunk 1]
```diff
@@ -108,9 +108,6 @@ func getElasticsearchCR(logging *logging.ClusterLogging, elasticsearchName strin
 		Roles:        []v1alpha1.ElasticsearchNodeRole{"client", "data", "master"},
```
Comment (josefkarasek) [inline pkg/k8shandler/logstore.go]: For the purpose of Cluster logging, we only allow CDM nodes?
If yes, then this PR is good.

**Comment by josefkarasek** [comment]
> We still have the ability to define different Resource requirements on different nodes

I think I wrote this days ago and it got published now, when I clicked at *submit review* :-1:

**Comment by ewolinetz** [comment]
> For the purpose of Cluster logging, we only allow CDM nodes?

Correct, for now this is the only scenario we support when it comes from CLO. 

> We still have the ability to define different Resource requirements on different nodes

With CLO we are only going to expose the ability to configure the resources for all nodes at this time...

**Pull request merged**